docs/upgrade.md

```diff
@@ -6,202 +6,133 @@ sidebar_label: Upgrade
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-operator.yaml
+./upgrade_sc.sh
 ```
 
-The output must be similar to the following.
+Alternatively, you can skip this step and re-apply your StorageClasses as per the 0.7.0 volume policy specification. 
 
-```
-namespace "openebs" created
-serviceaccount "openebs-maya-operator" created
-clusterrole "openebs-maya-operator" created
-clusterrolebinding "openebs-maya-operator" created
-deployment "maya-apiserver" created
-service "maya-apiserver-service" created
-deployment "openebs-provisioner" created
-deployment "openebs-snapshot-operator" created
-customresourcedefinition "storagepoolclaims.openebs.io" created
-customresourcedefinition "storagepools.openebs.io" created
-storageclass "openebs-standard" created
-storageclass "openebs-snapshot-promoter" created
-customresourcedefinition "volumepolicies.openebs.io" created
-```
+**Note: StorageClasses have to updated prior to provisioning any new volumes with 0.7.0.**
 
-You must deploy the 0.6 storage classes using the following command.
+### Upgrading OpenEBS Operator CRDs and Deployments
 
-```
-kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-storageclasses.yaml
-```
+The upgrade steps vary depending on the way OpenEBS was installed, select one of the following:
 
-The output must be similar to the following.
+#### Install/Upgrade using kubectl (using openebs-operator.yaml )
 
-```
-storageclass "openebs-standalone" created
-storageclass "openebs-percona" created
-storageclass "openebs-jupyter" created
-storageclass "openebs-mongodb" created
-storageclass "openebs-cassandra" created
-storageclass "openebs-redis" created
-storageclass "openebs-kafka" created
-storageclass "openebs-zk" created
-storageclass "openebs-es-data-sc" created
-```
+**The sample steps below will work if you have installed openebs without modifying the default values in openebs-operator.yaml. If you have customized it for your cluster, you will have to download the 0.7.0 openebs-operator.yaml and cutomize it again**
 
-**Upgrade Using Helm** ( OpenEBS was installed by helm chart)
-
-**Note:** The following sample steps will work if you have installed OpenEBS with default values provided by stable/openebs helm chart.
-
-Run the following command to view the release name.
-
-```
-helm ls
 ```
+# Starting with OpenEBS 0.6, all the components are installed in namespace `openebs`
+# as opposed to `default` namespace in earlier releases.
 
-The output is similar to the following.
+kubectl delete -f https://raw.githubusercontent.com/openebs/openebs/v0.6/k8s/openebs-operator.yaml
 
-```
-NAME    REVISION        UPDATED                         STATUS          CHART           NAMESPACE
-openebs 1               Fri Aug  3 14:50:11 2018        DEPLOYED        openebs-0.5.3   openebs
-```
-
-Here the release name is openebs.
+#Wait for objects to be delete, you can check using `kubectl get deploy`
```

Comment:
Wait for the objects to delete. Check if they are deleted using the `kubectl get deploy` command.